src/OSPSuite.UI/Services/ImageListRetriever.cs

```diff
@@ -40,7 +40,7 @@ public ImageCollection AllImages24x24
          }
       }
 
-      public ImageCollection AllImages32x32
+      public SvgImageCollection AllImages32x32
```

Comment:
Same question: why there is no **AllImages48x48** ?

Reply:
I have no idea. I just ported the one code to the other.

Reply:
I have exported 48x48 eventhough this is not used anywhere directly